### PR TITLE
Re-split the test buckets based on latest test runtime data

### PR DESCRIPTION
**Pull request by blindpirate**

### CODE REVIEW

**Comment by blindpirate** [comment]
@bot-gradle test and merge

File: .teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt [hunk 1]
```diff
@@ -153,7 +153,7 @@ class StatisticBasedFunctionalTestBucketProvider(private val model: CIBuildModel
     }
 
     private fun determineSubProjectClassTimes(testCoverage: TestCoverage, buildProjectClassTimes: BuildProjectToSubprojectTestClassTimes): Map<String, List<TestClassTime>>? {
-        val testCoverageId = testCoverage.asId("Gradle_Check")
+        val testCoverageId = testCoverage.asId("Gradle_Master_Check")
```
Comment (wolfs) [inline .teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt:156]: Can you extract the configuration name to a constant?

**Review by wolfs — APPROVED** [review]
LGTM!

**Comment by wolfs** [comment]
@bot-gradle test and merge

File: .teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt (hunk 1 above)
Comment (blindpirate) [reply .teamcity/src/main/kotlin/model/FunctionalTestBucketProvider.kt:156]: Followed up here: https://github.com/gradle/gradle/pull/17505

**Pull request merged**